dash/webapp/pages/toggles.js

```diff
@@ -16,82 +17,34 @@ const Button = styled.button`
   margin-right: 18px;
 `;
 
+const Status = styled.div`
+  width: 10px;
+  height: 10px;
+  margin-left: 10px;
+  border-radius: 50%;
+  background: ${props => (props.active ? 'green' : 'red')};
```

Comment:
I can't tell the difference between these – just sayin'.

Reply:
Changed to use words and lightgrey / green.